### ISSUE TITLE
Change the Twitter logo

### ISSUE THREAD

**Issue by mathlover2** [twitter/the-algorithm#1106]
**Is your feature request related to a problem? Please describe.**
The old blue bird logo doesn't match the new owner.

**Describe the solution you'd like**
I want a still image of Eric Cartman trying to fake his way into the Special Olympics to be the new Twitter logo. Alternative, that meme of Steve Buscemi trying to pass himself off as cool, but with a MAGA hat and Musk's face instead.

**Describe alternatives you've considered**
I considered Beavis and Butthead, a butt, a poop emoji, an emerald mine, Musk's lonely beside table with those stupid fake guns, that photo with Musk going to a Halloween party with his mother, Musk in Greek clothing with the words "Oedipus Rex" above his head, and 100 other different ideas, but no other images besides those two seem to capture the puerile willingness of Musk to pander to crypto bros and far-right jackasses for his own potential benefit. 

**Additional context**
In all seriousness: my apologies to anyone that isn't kissing up to Musk who is stuck with having to moderate this GH repository. If you feel down about this, just remember that the mockery is 100% **not** aimed at you, but at your company's insane CEO. I'd strongly suggest resigning and going elsewhere if you can: he's going to drive the site into the ground soon, and honestly any job, even outside of tech, is better than working for Elon Musk.

**Comment by mathlover2** [comment]
The following video may also work as a new logo. Musk should make not seeing it a perk of subscribing to Twitter Blue.

(I am going to hell for posting this here.)

https://www.youtube.com/watch?v=SAZzpaWpGIA

**Comment by TruncatedDinosour** [comment]
> https://www.youtube.com/watch?v=SAZzpaWpGIA

i think tina would work as a better logo, #871

**Comment by mathlover2** [comment]
True. I'm trying to give Musk an aneurysm, figuratively speaking. 

Cats make good logos, but that video makes for Musk having a conniption that shortens his life by one month.

**Comment by ondra05** [comment]
LGTM, birds aren't real, after all.
